src/mlpack/methods/logistic_regression/logistic_regression_main.cpp

```diff
@@ -102,9 +102,8 @@ BINDING_LONG_DESC(
     PRINT_PARAM_STRING("output_probabilities") +
     "\n\n"
     "This implementation of logistic regression does not support the general "
-    "multi-class case but instead only the two-class case.  Any labels must "
-    "be either 0 or 1.  For more classes, see the softmax_regression "
-    "program.");
+    "multi-class case but instead only the two-class case.  For more classes, "
```

Comment:
I guess you like to avoid to provide some explicit examples here, like R, Julia -> `[1, 2]`, C++ `[0, 1]`?

Reply:
Yeah, it would be really easy for this to go out of date, for instance if we added a new language we'd need to remember to update this too.  I'm actually working on another PR to update the R bindings such that they will throw an error if a zero value is passed for the labels.  I dunno, I'm not sure what the best way to go here is... I suppose it would be possible to add another documentation function to provide specific strings for each language here (like `PARAM_STRING()`, etc.), but I'm not sure it's worthwhile.

Reply:
Makes sense, my only concern is that the change made this a little bit more vague; a user might wonder does this mean `[0, 1]` or `[1, 2]`, or can I use `[5, 10]`. Perhaps it makes it more clear if we add a note that says something like it should follow the conventions of the language?